src/styles/dialog.scss

```diff
@@ -175,4 +175,12 @@ $button: #{$fd-namespace}-button;
       color: $fd-dialog-resize-handle-color;
     }
   }
+
+  &__no-backdrop:not(.#{$block}__targeted) {
+    position: fixed;
```

Comment:
why the position is fixed and not absolute?

Reply:
If it's absolute you can use the mixin we already have for this:
```
@mixin fd-absolute-center {
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
}
```

Reply:
Well, I decided to keep fixed because it centers the element in the view, not in the parent. Although in fundamental-ngx both fixed and absolute will work (because parent `fd-dialog-container` is attached directly to body), it feels reasonable to provide unified solution, that works well in all scenarios

Reply:
Awesome :) In this case please adjust the `fd-absolute-center` mixin to accept as parameter the position and reuse it in your code. 
You can do something like:
```
@mixin fd-absolute-center($position: absolute) {
  position: $position; 
  left: 50%;
  top: 50%; 
  transform: translate(-50%, -50%); 
}
```

Reply:
sure, done